src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -49,83 +49,83 @@
  */
 public final class RxJavaPlugins {
     @Nullable
-    static volatile Consumer<Throwable> errorHandler;
+    static volatile Consumer<? super Throwable> errorHandler;
 
     @Nullable
-    static volatile Function<Runnable, Runnable> onScheduleHandler;
+    static volatile Function<? super Runnable, ? extends Runnable> onScheduleHandler;
```

Comment:
I'm not too keen on these variance changes unless you can show that there is an inference problem with lambdas under 8 without them.